packages/svelteui-core/src/components/Table/Table.styles.ts

```diff
@@ -0,0 +1,42 @@
+import { createStyles } from '$lib/styles';
+import type { SvelteUINumberSize } from '$lib/styles';
+
+export interface TableStyleParams {
+	withBorder: boolean;
+	striped: boolean;
+	highlightOnHover: boolean;
+	cellPadding: SvelteUINumberSize;
+	textAlign: 'center' | 'left' | 'right';
+}
+
+export const padding = {
+	sm: 4,
+	md: 8,
+	lg: 16
+};
+
+export default createStyles(
+	(theme, { withBorder, highlightOnHover, striped, cellPadding, textAlign }: TableStyleParams) => {
+		const { themeColor } = theme.fn;
+		return {
+			root: {
+				borderCollapse: 'collapse',
+				cursor: highlightOnHover ? 'pointer' : 'default',
+				border: withBorder ? `1px solid ${themeColor('gray', 6)}` : '',
+				borderStyle: withBorder ? '' : 'hidden',
+				'& > * > th, & > * > td': {
```

Comment:
Hum, I would be specific here:

```suggestion
				'& > thead > tr > th, & > tbody > tr > td': {
```